duckduckgo/BrowserServicesKit
# Attribution pixel

Task/Issue URL: https://app.asana.com/0/414235014887631/1203017403159775/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1862
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1388
What kind of version bump will this require?: Major

**Description**:
Class to track and aggregate the number of page loads that have had an active exemption within a specified time period.

**Steps to test this PR**:
1. Run tests
1. Visit attribution sites - make sure the pageLoadsCount increases by 1
1. Go back and forward, visit the same vendor and notice the counter increases whenever we are in scope of the same domain.
1. Visit another vendor or open new tab and visit same vendor site - the counter should increase
1. Change the sendInterval const to notice that pixels is sent and the stats are reseted.
